README.adoc

```diff
@@ -52,7 +52,11 @@ http://maven.apache.org/plugins/maven-deploy-plugin/examples/deploy-ssh-external
 
 You can alternatively use a docker container to build the website :
 
-    $ docker build -t james/site dockerfiles/site
+You need to build the homepage by:
```

Comment:
Because Quynh had problems merging both.

We decided in daily to accept two dockers.